CMakeLists.txt

```diff
@@ -18,11 +18,12 @@ else()
   set(NANOGUI_USE_GLAD_DEFAULT OFF)
 endif()
 
-option(NANOGUI_BUILD_EXAMPLE "Build NanoGUI example application?" ON)
-option(NANOGUI_BUILD_SHARED  "Build NanoGUI as a shared library?" ON)
-option(NANOGUI_BUILD_PYTHON  "Build a Python plugin for NanoGUI?" ON)
-option(NANOGUI_USE_GLAD      "Use Glad OpenGL loader library?" ${NANOGUI_USE_GLAD_DEFAULT})
-option(NANOGUI_INSTALL       "Install NanoGUI on `make install`?" ON)
+option(NANOGUI_BUILD_EXAMPLE         "Build NanoGUI example application?" ON)
+option(NANOGUI_BUILD_SHARED          "Build NanoGUI as a shared library?" ON)
+option(NANOGUI_BUILD_PYTHON          "Build a Python plugin for NanoGUI?" ON)
+option(NANOGUI_USE_GLAD              "Use Glad OpenGL loader library?" ${NANOGUI_USE_GLAD_DEFAULT})
+option(NANOGUI_INSTALL               "Install NanoGUI on `make install`?" ON)
+option(NANOGUI_DEFAULT_GL_VERSION    "Default to GL3 compatability?"      ON)
```

Comment:
Verbiage on option string is slightly confusing, what is being controlled here is nano*vg* version.  Maybe `"Use NanoVG OpenGL 3.x+ definitions"` or something?  The 3+ will work for OpenGL 4, which is why `GL3` is slightly misleading.

Or maybe, at the cost of extreme verbosity, change this to be `NANOGUI_DEFAULT_NANOVG_VERSION` for extra clarity?